src/cloud/env/environments-ssh.md

```diff
@@ -118,20 +118,13 @@ You need the following requirements to sFTP into cloud environments:
 *  You need to use a client that supports SSH key authentication for sFTP and use your SSH public key.
 *  Your public SSH key must be added to the target environment. For Starter environments and Pro Integration environments, you can add it through the Project Web Interface. For Pro Staging and Production, you must enter a [Support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) with your public key attached. **Never provide your private SSH key.**
 
-When configuring sFTP, use your SSH public key and the following information for access:
+When configuring sFTP, use your SSH access destination (see [SSH to an Environment](#ssh), example `<project-id-name>@<cloud-host>`) and the following information for access:
 
-*  Username: All content before the `@` in your public SSH key.
+*  Username: All content before the `@` in your SSH access destination.
```

Comment:
I don't see where the Project UI or the Cloud guide has used the word destination to describe the SSH access command.

```suggestion
*  Username: All content before the `@` in your SSH access.
```

Reply:
Agreed. I used the term _destination_ here because it's the term used by the ssh man pages. My primary concern is fixing the incorrect instruction to use parts of 'your public SSH key' which really doesn't make any technical sense.